### PR TITLE
Don't let users submit blank fields in InputList

### PR DESCRIPTION
**Pull request by adriana-alter**
Resolves https://jira.nypl.org/browse/SIMPLY-1797

### CODE REVIEW

**Comment by adriana-alter** [comment]
That was my first idea, but when I tried it out I thought it made the form look broken in a way that might throw users.  Might still be worth it though.  What do you think?

**Comment by aslagle** [comment]
Hard to say without trying it, but having nothing happen when you click the button could also make the form seem broken.

**Comment by adriana-alter** [comment]
That's a very good point.  I'll try disabling it again!

File: src/components/InputList.tsx [hunk 1]
```diff
@@ -69,7 +73,7 @@ export default class InputList extends React.Component<InputListProps, InputList
           <button
             type="button"
             className="btn btn-default add-list-item"
-            disabled={this.props.disabled}
+            disabled={this.props.disabled || !this.state.newItem.length}
```
Comment (EdwinGuzman) [inline src/components/InputList.tsx]: This seems fine. When I add one value and then add it, the next input field will have a button that _looks_ enabled but when the input field is empty and you try to click the add button, it doesn't work (which is good).

Comment (adriana-alter) [reply src/components/InputList.tsx]: It definitely should not look enabled; good catch.  Will fix.

**Pull request merged**